docs/images/bluefin/developer-experience.md

```diff
@@ -1,7 +1,17 @@
 # bluefin-dx - The Bluefin Developer Experience
 
-Dedicated developer image with bundled tools. It endevaours to be the world's most powerful cloud native developer environment. :) It includes everything in the base image plus the following tools:
+Dedicated developer image with bundled tools. It endevaours to be the world's most powerful cloud native developer environment. :) It includes everything in the base image plus a curated set of developer tools and convenience shortcuts..
```

Comment:
Do you want the double period ".." at the end of this sentence? Seems like it should be 3 (and ellipsis) or else a single one.

Reply:
Oh, it was `just` a typo. Get it?